nexus/types/src/deployment/blueprint_display.rs

```diff
@@ -176,6 +192,9 @@ pub trait BpSledSubtableData {
 }
 
 /// A table specific to a sled resource, such as a zone or disk.
+//
+// TODO: Rename to `BpTable` since it is also used for blueprint wide tables
+// like those relating to `ClickhouseClusterConfig`?
```

Comment:
I'd vote yes on this, I think? Nothing in this table is sled-specific, AFAICT; it's more like `BpGenerationTable` or something.

Reply:
> I'd vote yes on this, I think? Nothing in this table is sled-specific, AFAICT; it's more like `BpGenerationTable` or something.

Done in [f0ae7db](https://github.com/oxidecomputer/omicron/pull/6968/commits/f0ae7dbbdf3c8b406a48ba0e19e6db7857c683f6)

I thought this was going to be a larger change. I liked `BpGenerationTable`, but went with the shorter `BpTable` because of the ripple of changes to other type names.